src/languages/de_DE.json

```diff
@@ -142,7 +142,7 @@
     "VLD_906": "Das Preset enthält ein verstecktes BAN gefolgt von einem PICK, ohne dass das BAN vorher aufgedeckt wird",
     "VLD_907": "Das Preset enthält einen versteckten Zug, der nie aufgedeckt wird",
     "VLD_908": "Das Preset enthält keine Züge",
-    "VLD_909": "Die angegebene presetId ist ungültig"
+    "VLD_909": "Die angegebene preset id ist ungültig"
```

Comment:
This is a technical "you are apparently fiddling with the application, but you're doing it wrong" error message, so I think keeping it as `presetId` here is better

Reply:
Got it. Will revert.